task/platform_adapter.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/influxdata/influxdb/task/options"
 )
 
+// TODO(lh): A temporary error response for all finds that use user id's
+// This should be removed when we transition to the new kv store.
+var ErrTempDisabled = errors.New("temporarily disabled")
```

Comment:
We started but didn't resolve an offline conversation about this.

We shouldn't be using this error. The platform adapter should instead accept a UserResourceMappingService, and handle the User filter case by looking up that user's tasks and then calling `FindTaskByID` on each individual task.

We will refactor that to a more efficient approach as part of the proposed refactor in https://github.com/influxdata/influxdb/issues/11063.

Reply:
I will make that change. It should be a clean change and allow it to be a little more permanent then the current solution.